packages/expo/scripts/resolveAppEntry.js

```diff
@@ -29,12 +29,18 @@ if (!platform || !projectRoot) {
   process.exit(1);
 }
 
+// Prevent any logs from the app.config.js
+// from being used in the output of this command.
+const originalWrite = process.stdout.write;
+process.stdout.write = () => {};
+
+// Resolve the entry point
 const entry = resolveEntryPoint(projectRoot, { platform });
 
+// Restore stdout
+process.stdout.write = originalWrite;
```

Comment:
If we decide to go with this approach, I have a few suggestions:

1. Wrap this code in try-finally so that we restore stdout at the end no matter what
2. Consider replacing process.stdout with a new stream object instead of overwriting its write() method
3. Consider sending stdout to stderr (`process.stdout = process.stderr`)). Is it more confusing for console.log() output to be hidden, or for it to go to stderr?